### PR TITLE
[6.14.z] Sync requirements with master branch

### PR DESCRIPTION
**Pull request by ogajduse**
Sync with master commit
42a058ae13a80a461306a867412e37acae14197f

### CODE REVIEW

**Comment by Gauravtalreja1** [comment]
trigger: test-robottelo

File: requirements.txt [hunk 1]
```diff
@@ -1,16 +1,16 @@
 # Version updates managed by dependabot
 
 betelgeuse==1.10.0
-broker[docker]==0.3.2
-cryptography==41.0.0
-deepdiff==6.3.0
-dynaconf[vault]==3.1.12
+broker[docker]==0.3.3
```
Comment (Gauravtalreja1) [inline requirements.txt:4]: @ogajduse Can you update this PR to pull missed cherrypicks from requirements-optional.txt?

**Review by jyejare — APPROVED** [review]

**Comment by ogajduse** [comment]
trigger: test-robottelo

**Comment by ogajduse** [comment]
trigger: test-robottelo
pod_resources_size: extra_large

**Review by jameerpathan111 — APPROVED** [review]

**Pull request merged**